activesupport/lib/active_support/error_reporter.rb

```diff
@@ -276,7 +276,7 @@ def report(error, handled: true, severity: handled ? :warning : :error, context:
 
     private
       def ensure_backtrace(error)
-        return if error.frozen? # re-raising won't add a backtrace
+        return if error.frozen? # re-raising won't add a backtrace or set the cause
```

Comment:
Should I update the comment on lines 283-286?

There's no way to set the cause besides the `raise`+`rescue` trick, so Ruby 3.4's `set_backtrace` trick won't help us here.

Reply:
Yes, you can mostly remove it I think.